src/index.html

```diff
@@ -4,9 +4,359 @@
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div class="card" data-qa="card">
+        <div class="card__content">
+          <img
+            src="./images/imac.jpeg" alt="iMac"
+            class="card__img"
+          >
+          <p class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+          <p class="card__product-code">
+            Product code: 195434
+          </p>
+          <div class="card__review">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__review-count">
+              <p>Reviews: 5</p>
+            </div>
+          </div>
+          <div class="card__price">
+            <p class="card__price-text">Price:</p>
+            <p class="card__price-number">$2,199</p>
+          </div>
+          <a
+            href="#"
+            data-qa="card-hover"
+            class="btn btn--buy"
+          >
+            buy
+          </a>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card__content">
+          <img
+            src="./images/imac.jpeg" alt="iMac"
+            class="card__img"
+          >
+          <p class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+          <p class="card__product-code">
+            Product code: 195434
+          </p>
+          <div class="card__review">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__review-count">
+              <p>Reviews: 5</p>
+            </div>
+          </div>
+          <div class="card__price">
+            <p class="card__price-text">Price:</p>
+            <p class="card__price-number">$2,199</p>
+          </div>
+          <a
+            href="#"
+            class="btn btn--buy"
+          >
+            buy
+          </a>
+        </div>
+      </div>
+      <div class="card">
```

Comment:
Add a blank line between the cards for better readability
```suggestion
      </div>
      
      <div class="card">
```